runtime/firecracker-runc-config.json.example

```diff
@@ -0,0 +1,133 @@
+{
+    "ociVersion": "1.0.1-dev",
+    "process": {
+        "terminal": false,
+        "user": {
+            "uid": 0,
+            "gid": 0
+        },
+        "args": [
+            "/firecracker",
+            "--api-sock",
+            "api.socket"
+        ],
+        "env": [
+            "PATH=/"
+        ],
+        "cwd": "/",
+        "capabilities": {
+            "effective": [
+            ],
+            "bounding": [
+            ],
+            "inheritable": [
+                "CAP_NET_ADMIN"
+            ],
+            "permitted": [
+                "CAP_NET_ADMIN"
+            ],
+            "ambient": [
+                "CAP_NET_ADMIN"
+            ]
+        },
+        "rlimits": [
+            {
+                "type": "RLIMIT_NOFILE",
+                "hard": 1024,
+                "soft": 1024
+            }
+        ],
+        "noNewPrivileges": true
+    },
+    "root": {
+        "path": "rootfs",
+        "readonly": false
+    },
+    "hostname": "runc",
+    "mounts": [
+        {
+            "destination": "/proc",
+            "type": "proc",
+            "source": "proc"
+        }
+    ],
+    "linux": {
+        "devices": [
+            {
+               "path": "/dev/kvm",
+               "type": "c",
+               "major": 10,
+               "minor": 232,
+               "fileMode": 438,
+               "uid": 0,
+               "gid": 0
+            },
+            {
+               "path": "/dev/net/tun",
+               "type": "c",
+               "major": 10,
+               "minor": 200,
+               "fileMode": 438,
+               "uid": 0,
+               "gid": 0
+            }
+        ],
+        "resources": {
+            "devices": [
+                {
+                    "allow": false,
+                    "access": "rwm"
+                },
+                {
+                    "allow": true,
+                    "major": 10,
+                    "minor": 232,
+                    "access": "rwm"
+                },
+                {
+                    "allow": true,
+                    "major": 10,
+                    "minor": 200,
+                    "access": "rwm"
+                }
+            ]
+        },
+        "namespaces": [
+            {
+                "type": "cgroup"
+            },
+            {
+                "type": "pid"
+            },
+            {
+                "type": "network"
+            },
+            {
+                "type": "ipc"
+            },
+            {
+                "type": "uts"
+            },
+            {
+                "type": "mount"
+            }
+        ],
+        "maskedPaths": [
+            "/proc/kcore",
+            "/proc/latency_stats",
+            "/proc/timer_list",
+            "/proc/timer_stats",
+            "/proc/sched_debug",
+            "/sys/firmware",
+            "/proc/scsi"
+        ],
+        "readonlyPaths": [
+            "/proc/asound",
```

Comment:
Really? Music? https://alsa.opensrc.org/Proc_asound_documentation

Reply:
Sadly yes, it leaks information due to not being namespaced by the kernel: https://github.com/moby/moby/issues/38285

So no music in our jail :-(

But actually I do think we need to move it `maskedPaths` rather than `readonlyPaths` based on that docker thread (which differs from runc defaults).

Reply:
:+1: I think `maskedPaths` makes more sense for this one.